.github/workflows/releases.yml

```diff
@@ -0,0 +1,40 @@
+name: Releases
+
+on: push
+
+jobs:
+  releases:
+    runs-on: ubuntu-latest
+    timeout-minutes: 60
+    steps:
+    - uses: actions/checkout@v3
+    - uses: cachix/install-nix-action@v22
```

Comment:
should probably pin commit hash instead of `v22`; same for `actions/checkout@v3` although it probably comes from github itself?